code/modules/sanity/sanity_mob.dm

```diff
@@ -153,7 +153,7 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 
 /datum/sanity/proc/handle_view()
 	. = 0
-	activate_mobs_in_range(owner, SANITY_MOB_DISTANCE_ACTIVATION)
+	activate_mobs_in_range(owner, world.view)
```

Comment:
This isn't an optimization, it just reduces the search area, which might be undesirable.

Reply:
Can we redefine SANITY_MOB_DISTANCE_ACTIVATION to world.view instead of its current 12 instead?